### PR TITLE
SilverCore research for the silver golem

### PR DESCRIPTION
**Pull request by FluffiestFloof**
## About the PR

Silver Golem not included.
Neither are the rest of the research items related to this one like QSI and HandTeleporter.
Includes all the golem sprites tho

## Media

- [ [X] ] :trollface: I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase 

## Breaking changes

**Changelog**

### CODE REVIEW

**Comment by DEATHB4DEFEAT** [comment]
\> "no golem"
\> look inside
\> golem

:trollface:

File: Resources/Prototypes/Entities/Mobs/Species/golem.yml
Comment (DEATHB4DEFEAT) [inline Resources/Prototypes/Entities/Mobs/Species/golem.yml:1]: gone, reduced to comments

File: Resources/Prototypes/Entities/Objects/Specific/Research/silvercore.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+- type: entity
+  parent: BaseItem
+  id: CoreSilver
+  name: silver alloy core
+  description: Syntetic alloy made of silver and other materials.
```
Comment (DEATHB4DEFEAT) [inline Resources/Prototypes/Entities/Objects/Specific/Research/silvercore.yml:5]:
```suggestion
  description: A synthetic alloy made of silver and other materials.
```

File: Resources/Prototypes/Nyanotrasen/Recipes/Lathes/bluespace.yml [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+#   icon:
+#     sprite: Nyanotrasen/Objects/Devices/QSI.rsi
+#     state: icon
+#   result: QSI
+#   completetime: 5
+#   materials:
+#     Bluespace: 25
+#     Glass: 250
+#     Plastic: 250
+
```
Comment (DEATHB4DEFEAT) [inline Resources/Prototypes/Nyanotrasen/Recipes/Lathes/bluespace.yml:12]: Unrelated
```suggestion
```

File: Resources/Prototypes/Nyanotrasen/Recipes/Lathes/bluespace.yml [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+#   id: TrashBagHolding
+#   icon:
+#     sprite: Objects/Specific/Janitorial/trashbag.rsi
+#     state: blue-icon-0
+#   result: TrashBagHolding
+#   completetime: 4
+#   materials:
+#     Bluespace: 200
+#     Durathread: 750
+#     Plastic: 200
```
Comment (DEATHB4DEFEAT) [inline Resources/Prototypes/Nyanotrasen/Recipes/Lathes/bluespace.yml:60]: Unrelated
```suggestion
```

File: Resources/Prototypes/Nyanotrasen/Research/experimental.yml [hunk 4]
```diff
@@ -16,16 +16,16 @@
...
+  icon:
+    sprite: Nyanotrasen/Objects/Devices/QSI.rsi
+    state: icon
+  discipline: Experimental
+  tier: 3
+  cost: 15000
+  recipeUnlocks:
+  - CoreSilver
+  # - HandTeleporter
+  # - QSI
```
Comment (DEATHB4DEFEAT) [inline Resources/Prototypes/Nyanotrasen/Research/experimental.yml:31]:
```suggestion
```

File: Resources/Textures/Nyanotrasen/Objects/Devices/QSI.rsi/icon.png
Comment (DEATHB4DEFEAT) [inline Resources/Textures/Nyanotrasen/Objects/Devices/QSI.rsi/icon.png:1]: Unrelated

File: Resources/Textures/Nyanotrasen/Objects/Devices/QSI.rsi/meta.json
Comment (DEATHB4DEFEAT) [inline Resources/Textures/Nyanotrasen/Objects/Devices/QSI.rsi/meta.json:1]: Unrelated

**Review by DEATHB4DEFEAT — DISMISSED** [review]

File: Resources/Textures/Nyanotrasen/Objects/Devices/QSI.rsi/icon.png
Comment (FluffiestFloof) [reply Resources/Textures/Nyanotrasen/Objects/Devices/QSI.rsi/icon.png:1]: Entirely related.

Comment (DEATHB4DEFEAT) [reply Resources/Textures/Nyanotrasen/Objects/Devices/QSI.rsi/icon.png:1]: I have been bested

**Review by Colin-Tel — APPROVED** [review]
I love comments!

**Comment by DebugOk** [comment]
@DEATHB4DEFEAT Please re-review and change it if applicable

**Comment by DebugOk** [comment]
Considering it seems to work fine, and I dont see why we should keep this waiting any longer, I'll just merge it without waiting for death to re-review :clueless:
